task/deprecated-image-check/0.4/MIGRATION.md

```diff
@@ -0,0 +1,8 @@
+# Migration from 0.3 to 0.4
+
+The result `PYXIS_HTTP_CODE` from `init` task was removed.
```

Comment:
Are you sure that this result is not used in other (like EC) pipelines?

Reply:
it isn't used in whole repo